frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -414,6 +415,7 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
         ],
         filters: [
             props.filters ?? getDefaultFilters(props.personUUID),
+            { persist: true, prefix: `${getCurrentTeamId()}__${key}` },
```

Comment:
persist by logic key and team, since we don't want to show filters for playlist X on playlist Y